screenpy_selenium/target.py

```diff
@@ -35,6 +35,7 @@ class Target:
 
     @property
     def target_name(self):
+        """return the description when set or the 2nd half of the locator"""
```

Comment:
Nice catch!

Reply:
TBH, I think pylint caught that one.  Which is a good reason to have the linters running in github actions.  Once something sneaks by pre-commit it doesn't get scanned again.